pkg/cloud/cloud.go

```diff
@@ -78,16 +80,26 @@ var (
 	ErrAlreadyExists = errors.New("Resource already exists")
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
```

Comment:
nit: personally I'd prefer to move this into `pickRandomAvailabilityZone`, but that's OK if you want to leave it here.

Reply:
rand just need to be seeded once. so leave it at the init func.